resources/analysisTools/indelCallingWorkflow/canopyCluster_updatedFunction.R

```diff
@@ -1,7 +1,7 @@
 #
 # Copyright (c) 2018 German Cancer Research Center (DKFZ).
 #
-# Distributed under the GPL-3 License (license terms are at https://github.com/DKFZ-ODCF/IndelCallingWorkflow).
+# Distributed under the GPL-2+ License (license terms are at https://github.com/DKFZ-ODCF/IndelCallingWorkflow).
```

Comment:
2+ is this the used termin for this? I could not find it with a brief search.

Reply:
Also it is under dual license right?

Reply:
You'll find it by searching in Github.